curator/theme.json

```
@@ -58,6 +58,9 @@
 			]
 		},
 		"custom": {
+			"color": {
+				"link-background": "#F4ECEC"
```

Can we map this to the theme preset color variable instead? This way, if the user changes the theme's palette via global styles there, it will match.

```suggestion
				"link-background": "var(--wp--preset--color--tertiary)"
```

---

Makes sense.
Tertiary color in the figma seems to be using the color F4ECEC in the rectangle, but wrongly indicated a different lighter color in text (FCFAFA). Updated tertiary and used the same for link background.